publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelper.java

```diff
@@ -78,9 +78,13 @@ public void rabbitMqPropertiesInit(String protocol) {
      * This method is used to set the values of protocol and initialize the RabbitMq properties
      * @param protocol name
      */
-    private void protocolInit(String protocol) {
+    private void protocolInit(String protocol){
         rabbitMqPropertiesMap.get(protocol).setProtocol(protocol);
-        rabbitMqPropertiesMap.get(protocol).init();
+        try {
+            rabbitMqPropertiesMap.get(protocol).init();
+        } catch (InvalidExchangeException e) {
+            log.error("InvalidExchangeException occurred ::" + e.getMessage());
```

Comment:
Can this error log be changed for better readability,

An exception occured while checking for Exchange, Reason :